docs/launch-arbitrum-chain/features/advanced/choose-arbos-version.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Why choose a specific ArbOS version on your Arbitrum chain'
+description: 'Learn about choosing which ArbOS version to use on your chain.'
+author: pete-vielhaber
+sme:
+content_type: configuration
+---
+
+Customizing the ArbOS version refers to modifying the Nitro codebase to introduce a new, version-controlled iteration of ArbOS (Arbitrum's operating system-like layer) that activates custom behaviors, features, or state changes in a way that maintains backward compatibility and determinism for fraud proofs. This is an advanced customization primarily for live chains, allowing developers to extend or alter the State Transition Function (STF)—the core logic for block production and state updates—while ensuring safe, non-disruptive upgrades. Unlike standard upgrades (e.g., to canonical versions like ArbOS 20 "Atlas"), customization involves creating intermediate or project-specific versions (e.g., ArbOS 32 as a fork of 31) to incorporate bespoke elements like new precompiles, EVM opcodes, or state variables, without conflicting with official releases. It's recommended for teams with expertise or partners (e.g., Rollup-as-a-Service providers), as it requires audits, maintenance, and careful handling to avoid issues like chain re-orgs or failed fraud proofs.
```

Comment:
The title is about choosing one's Arbos version, but the content is mostly about customizing Arbos.